packages/xo-web/src/common/intl/messages.js

```diff
@@ -950,6 +950,8 @@ const messages = {
   smartRebootHostLabel: 'Smart reboot',
   smartRebootHostTooltip: 'Suspend resident VMs, reboot host and resume VMs automatically',
   forceRebootHostLabel: 'Force reboot',
+  forceSmartRebootHost:
+    '{nVms, number} VM{nVms, plural, one {} other {s}} have suspended operation blocks. Would you like to force?',
```

Comment:
```suggestion
    'Smart Reboot failed because {nVms, number} VM{nVms, plural, one {} other {s}} ha{nVms, plural, one {s} other {ve}} {nVms, plural, one {its} other {their}} Suspend operation blocked. Would you like to force?',
```